files/install-requirements.yaml

```diff
@@ -35,6 +35,11 @@
     pip:
       name: git+https://github.com/packit-service/sandcastle
 
+  - name: Pip install ogr from master
```

Comment:
I'm -1 on this since we have the matrix now.

Reply:
Sure. I done this, just to see if there is no difference with zuul. Because there were fails